trieste/acquisition/function.py

```diff
@@ -1293,8 +1298,7 @@ def update_acquisition_function(
 class batch_monte_carlo_expected_improvement(AcquisitionFunctionClass):
     def __init__(self, sample_size: int, model: ProbabilisticModel, eta: TensorType, jitter: float):
         """
-
-        :param sampler:  BatchReparametrizationSampler.
+        :param sampler:  ReparametrizationSampler.
```

Comment:
you do need a batch sampler here, right?

Reply:
Actually not. This is a type guide. BatchReparameterizationSampler is the one specific to Gaussian models. I will rename our BatchReparameterizationSampler sampler to GaussianBatchReparameterizationSampler